### ISSUE TITLE
[Bug Report] Failure starting using `--config` flag

### ISSUE THREAD

**Issue by flying-sausages** [hkalexling/Mango#53]
**Describe the bug**
When passing `--config` flag, the binary returns the following error

**To Reproduce**
1. Download binary
2. Create config file
3. Run `mango --config="/path/to.yml"`

**Expected behavior**
Start up 0.5.0 with the premade config

**Environment (please complete the following information):**
 - OS:U20.04
 - Mango Version 0.5.0

**Docker (if you are running Mango in a Docker container)**
- nope

**Additional context**
```
/opt/mango/mango.bin --config="/opt/mango/config.yml"
Unhandled exception: Invalid option: --config=/opt/mango/config.yml (OptionParser::InvalidOption)
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
```

**Comment by flying-sausages** [comment]
This however works
`/opt/mango/mango.bin admin user list --config=/opt/mango/config.yml`

**Comment by hkalexling** [comment]
Fixed in [v0.5.1](https://github.com/hkalexling/Mango/releases/tag/v0.5.1). Thanks for the bug report!

**Comment by flying-sausages** [comment]
Works like a charm now, thanks a bunch!
